### PR TITLE
Scripts/Commands: Update reload commands names

### PR DESCRIPTION
**Pull request by ChazyTheBest**
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

**Changes proposed:**

- Update reload commands names

**Target branch(es):** 3.3.5/master

- [x] 3.3.5

**Issues addressed:** None

**Tests performed:** (Does it build, tested in-game, etc.)

- [x] Builds

### CODE REVIEW

File: src/server/scripts/Commands/cs_reload.cpp [hunk 1]
```diff
@@ -993,7 +993,7 @@ class reload_commandscript : public CommandScript
     {
         TC_LOG_INFO("misc", "Re-Loading Locales Achievement Reward Data...");
         sAchievementMgr->LoadRewardLocales();
-        handler->SendGlobalGMSysMessage("DB table `locales_achievement_reward` reloaded.");
+        handler->SendGlobalGMSysMessage("DB table `achievement_reward` reloaded.");
```
Comment (tkrokli) [inline src/server/scripts/Commands/cs_reload.cpp]: Isn't the new table name `achievement_reward_locale` ?

Comment (ChazyTheBest) [inline src/server/scripts/Commands/cs_reload.cpp]: yup, missed that one.

**Comment by tkrokli** [comment]
Maybe you could take a look at all the lines saying `TC_LOG_INFO("misc", "Re-Loading Locales [...]` and move the word `Locales` (in the form of `Locale`) to after the table name?

**Comment by ChazyTheBest** [comment]
Sure.

**Pull request merged**